library/ecp.c

```diff
@@ -2004,8 +2004,10 @@ static unsigned char ecp_pick_window_size( const mbedtls_ecp_group *grp,
      * Make sure w is within bounds.
      * (The last test is useful only for very small curves in the test suite.)
      */
+#if( MBEDTLS_ECP_WINDOW_SIZE < 5 )
```

Comment:
`w` can be 4, 5 or 6. If `w` is 6 and `MBEDTLS_ECP_WINDOW_SIZE` is 5 then the code used to set `w` to 5 but now it doesn't. The test should be `MBEDTLS_ECP_WINDOW_SIZE <= 5` or `MBEDTLS_ECP_WINDOW_SIZE < 6`.

Reply:
Ah, I missed the `w++`, thanks!